### ISSUE TITLE
Failed to activate the linter-js-yaml package

### ISSUE THREAD

**Issue by victtsl** [AtomLinter/linter-js-yaml#145]
Getting error below when launching atom/

```
Failed to activate the linter-js-yaml package
Unknown kind "!include" is specified for "!yaml" YAML type.
The error was thrown from the linter-js-yaml package. You can help by creating an issue. Please explain what actions triggered this error.
Create issue on the linter-js-yaml package
```

Stack trace from the error:
```
YAMLException: Unknown kind "!include" is specified for "!yaml" YAML type.
    at new Type (C:\Users\devuser\.atom\packages\linter-js-yaml\node_modules\js-yaml\lib\js-yaml\type.js:57:11)
    at file:///C:/Users/devuser/.atom/packages/linter-js-yaml/lib/linter-js-yaml.js:28:16
    at Array.map (<anonymous>)
    at file:///C:/Users/devuser/.atom/packages/linter-js-yaml/lib/linter-js-yaml.js:26:51
    at Config.observeKeyPath (C:\Users\devuser\AppData\Local\atom\app-1.31.2\resources\app\src\config.js:1042:11)
    at Config.observe (C:\Users\devuser\AppData\Local\atom\app-1.31.2\resources\app\src\config.js:485:25)
    at Object.activate (file:///C:/Users/devuser/.atom/packages/linter-js-yaml/lib/linter-js-yaml.js:25:40)
    at Package.activateNow (C:\Users\devuser\AppData\Local\atom\app-1.31.2\resources\app\src\package.js:230:33)
    at activationHookSubscriptions.add.packageManager.onDidTriggerActivationHook (C:\Users\devuser\AppData\Local\atom\app-1.31.2\resources\app\src\package.js:901:81)
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\devuser\AppData\Local\atom\app-1.31.2\resources\app\node_modules\event-kit\lib\emitter.js:27:20)
    at Emitter.module.exports.Emitter.emit (C:\Users\devuser\AppData\Local\atom\app-1.31.2\resources\app\node_modules\event-kit\lib\emitter.js:156:34)
    at PackageManager.triggerDeferredActivationHooks (C:\Users\devuser\AppData\Local\atom\app-1.31.2\resources\app\src\package-manager.js:718:40)
    at Promise.all.then (C:\Users\devuser\AppData\Local\atom\app-1.31.2\resources\app\src\package-manager.js:651:18)
    at <anonymous>
```

**Comment by yupylypenko** [comment]
Probably it could be usefull
(0 , _util.promisify) is not a function
Hide Stack Trace
TypeError: (0 , _util.promisify) is not a function
    at /home/ypylypenko/.atom/packages/linter-eslint/node_modules/sb-fs/lib/index.js:17:46
    at Array.forEach (native)
    at Object.<anonymous> (/home/ypylypenko/.atom/packages/linter-eslint/node_modules/sb-fs/lib/index.js:13:26)
    at Object.<anonymous> (/home/ypylypenko/.atom/packages/linter-eslint/node_modules/sb-fs/lib/index.js:51:3)
    at Module.get_Module._compile (/opt/atom/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/opt/atom/resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (file:///opt/atom/resources/app.asar/static/index.js:47:45)
    at require (/opt/atom/resources/app/src/native-compile-cache.js:66:33)
    at Object.<anonymous> (/home/ypylypenko/.atom/packages/linter-eslint/node_modules/atom-package-deps/lib/helpers.js:10:36)
    at Object.<anonymous> (/home/ypylypenko/.atom/packages/linter-eslint/node_modules/atom-package-deps/lib/helpers.js:196:3)
    at Module.get_Module._compile (/opt/atom/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/opt/atom/resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (file:///opt/atom/resources/app.asar/static/index.js:47:45)
    at require (/opt/atom/resources/app/src/native-compile-cache.js:66:33)
    at Object.<anonymous> (/home/ypylypenko/.atom/packages/linter/node_modules/atom-package-deps/lib/index.js:10:39)
    at Object.<anonymous> (/home/ypylypenko/.atom/packages/linter/node_modules/atom-package-deps/lib/index.js:55:3)
    at Module.get_Module._compile (/opt/atom/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/opt/atom/resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (file:///opt/atom/resources/app.asar/static/index.js:47:45)
    at require (/opt/atom/resources/app/src/native-compile-cache.js:66:33)
    at Object.activate (/home/ypylypenko/.atom/packages/linter-js-yaml/lib/linter-js-yaml.js:22:5)
    at Package.activateNow (/opt/atom/resources/app/src/package.js:230:33)
    at activationHookSubscriptions.add.packageManager.onDidTriggerActivationHook (/opt/atom/resources/app/src/package.js:901:81)
    at Function.module.exports.Emitter.simpleDispatch (/opt/atom/resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/opt/atom/resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at PackageManager.triggerActivationHook (/opt/atom/resources/app/src/package-manager.js:732:40)
    at Workspace.handleGrammarUsed (/opt/atom/resources/app/src/workspace.js:1269:38)
    at TextEditor.observeGrammar (/opt/atom/resources/app/src/text-editor.js:733:11)
    at onDidAddPaneItem (/opt/atom/resources/app/src/workspace.js:513:22)
    at Function.module.exports.Emitter.simpleDispatch (/opt/atom/resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/opt/atom/resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at PaneContainer.didAddPaneItem (/opt/atom/resources/app/src/pane-container.js:271:24)
    at Pane.addItem (/opt/atom/resources/app/src/pane.js:640:48)

**Comment by Arcanemagus** [comment]
@yupylypenko Your stack trace looks to either be from an _ancient_ version of Atom, or one of the hacked together broken versions that Arch installs.

**Comment by Arcanemagus** [comment]
@victtsl That _looks_ like something internal to `js-yaml`, are you still seeing this issue?

**Comment by victtsl** [comment]
No i'm good. Thanks.

**Comment by yupylypenko** [comment]
> @yupylypenko Your stack trace looks to either be from an _ancient_ version of Atom, or one of the hacked together broken versions that Arch installs.

Hello @Arcanemagus,

Thank you for assistance. It has been fixed after manually update to 1.37.0 deb on my Ubuntu from 1.35

So, the issue was in the Atom.
